services-js/registry-certs/client/store/BirthCertificateRequest.ts

```diff
@@ -53,6 +71,29 @@ export default class BirthCertificateRequest {
     this.currentStepCompleted = isCompleted;
   };
 
+  @action
+  public addVerificationStep = (): void => {
```

Comment:
Ah, ok.

I’d recommend replacing all of this with a `@computed` `steps` property that would return either `INITIAL_STEPS` or `INITIAL_STEPS_WITH_VERIFICATION`.

Reply:
Between this and your earlier comment, I definitely want to learn about `@computed`!